### DISCUSSION TITLE
Other brands of System

### DISCUSSION THREAD

**Discussion by dragonflyuk** [taligentx/dscKeybusInterface#71] [dscKeybusInterface Discussion]
Does anyone know if the wiring and protocols for alarm systems are consistent against other brands. Inparticular I have a teletek system that has wiring and voltages that look identical.

**Comment by taligentx** [comment]
Hi @dragonflyuk - as far as I have seen, the protocols are vendor-specific.  If the Teletek system is rebranded DSC equipment, you may be able to use this interface - the control boards and keypads would likely look similar.
